packages/core/src/external/carequality/ihe-gateway-v2/outbound/xca/process/schema.ts

```diff
@@ -0,0 +1,137 @@
+import { z } from "zod";
+import {
+  schemaOrEmpty,
+  schemaOrArray,
+  schemaOrArrayOrEmpty,
+  StringOrNumberSchema,
+} from "../../schema";
+
+const slot = z.object({
+  // even though this is called list, the IHE spec specifies its never a list: https://www.ihe-j.org/file2/docs/RFD/IHE_ITI_TF_R14_Vol3.pdf, Page 48
```

Comment:
Can you plz put this comment into a tsdoc? with the link under the `@see` field?